routers/api/v1/repo/issue_tracked_time.go

```diff
@@ -203,15 +206,61 @@ func ListTrackedTimesByRepository(ctx *context.APIContext) {
 	//   description: name of the repo
 	//   type: string
 	//   required: true
+	// - name: user
+	//   in: query
+	//   description: optional filter by user
+	//   type: string
+	// - name: created_after
+	//   in: query
+	//   description: filter by time witch tracked times were created after
+	//   type: string
+	// - name: created_before
```

Comment:
before  -  Only show notifications updated before the given time. This is a timestamp in ISO 8601 format: YYYY-MM-DDTHH:MM:SSZ